api/v1beta1/keystoneapi_types.go

```diff
@@ -119,6 +119,18 @@ type KeystoneAPISpecCore struct {
 	// TrustFlushSuspend - Suspend the cron job to purge trusts
 	TrustFlushSuspend bool `json:"trustFlushSuspend"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=1
+	// +kubebuilder:validation:Minimum=0
```

Comment:
should that be 1? I think setting it to 0 would create an infinite reconcile loop

Reply:
right, I set it to 1